src/main/java/com/yahoo/bullet/pubsub/PubSubMessage.java

```diff
@@ -21,11 +24,12 @@
  */
 @Getter
 public class PubSubMessage implements Serializable, JSONFormatter {
-    private static final long serialVersionUID = -5068189058170874687L;
     public static final Charset CHARSET = StandardCharsets.UTF_8;
+    private static final long serialVersionUID = 5096747716667851530L;
 
     private String id;
-    private byte[] content;
+    // Serializable enforced through the constructors, and getter. Is Object so GSON can reify an instance.
```

Comment:
Might want a setContent(Serializable) for serde usage

Reply:
Didn't have it before. I don't know if we want to be able to change the payload once created. I initially had things converting Strings to byte[] and if we want to add that kind of stuff back, we don't want anyone interfering with the payload.